packages/neo-one-client/src/user/keystore/LedgerKeyStore.ts

```diff
@@ -0,0 +1,232 @@
+import { AddressString, common, crypto } from '@neo-one/client-core';
+import { Monitor } from '@neo-one/monitor';
+import { utils } from '@neo-one/utils';
+import _ from 'lodash';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { distinctUntilChanged, map } from 'rxjs/operators';
+import { UnknownAccountError } from '../../errors';
+import { publicKeyToAddress } from '../../helpers';
+import { Account, NetworkType, UserAccount, UserAccountID, Witness } from '../../types';
+import { LedgerHandler } from './LedgerHandler';
+
+export interface Ledger {
+  readonly accKey: number;
```

Comment:
nit: bleh, abbreviations - use `accountKey` (assuming that's what `acc` stands for)